inc/class_captcha.php

```diff
@@ -307,6 +312,40 @@ function validate_captcha()
 				}
 			}
 		}
+		elseif(in_array($this->type, array(8)))
```

Comment:
Not much point in using `in_array` with a single element IMO.

```suggestion
		elseif($this->type == 8)
```